### PR TITLE
Make screenshots full width and solid BG

### PR DESCRIPTION
**Pull request by tofumatt**
Fix https://github.com/mozilla/addons.mozilla.org-mod/issues/4

### Before

screenshot 2016-03-19 16 29 16

### After

screenshot 2016-03-19 16 43 12

### CODE REVIEW

**Comment by wagnerand** [comment]
Nice screenshots, but somehow unrelated?

**Comment by tofumatt** [comment]
Wtf, I must've had them sorted wrongly. I'll fix it in a bit when I'm back on my mac.

**Comment by tofumatt** [comment]
Screenshots fixed.

**Comment by muffinresearch** [comment]
What does this look like with more items in the carousel? That would be a good comparison.

File: static/css/restyle.less [hunk 1]
```diff
@@ -1013,8 +1013,15 @@ button.good {
   color: @link-on-white-bg;
 }
 
-.previews {
+.restyle .previews {
   background: #f5f5f5;
+  margin-left: ~"calc(-1 * ((100vw - 100%) / 2))";
```
Comment (muffinresearch) [inline static/css/restyle.less]: This could use some comments.

**Comment by tofumatt** [comment]
screenshot 2016-03-21 13 16 13

With three screenshots. Fitting more in seems like a tonne of extra JS/weirdness work to be responsive... so three is much easier. But let me know if that's too awful.

File: static/css/restyle.less (hunk 1 above)
Comment (tofumatt) [inline static/css/restyle.less]: Comments added!

**Comment by muffinresearch** [comment]
On a large-ish resolution this looks a bit odd:

exquisite_stew____add-ons_for_firefox

I wonder if the carousel should be more limited to the content width but keep the background full-width?

Are there mocks for this?

**Comment by tofumatt** [comment]
Good call; I've updated it so the background is full width but the screenshots are kept centred, same `max-width` as the main page. Here's a crazy big window looking more normal:

screen shot 2016-03-22 at 12 59 21

**Comment by tofumatt** [comment]
Oh also I didn't see any mocks. @pwalm, any thoughts?

**Comment by pwalm** [comment]
Your most recent screenshot is bang on

**Comment by tofumatt** [comment]
Sweet! Thanks 😀

**Comment by muffinresearch** [comment]
I was still reviewing this and I was looking into the width calc. Seems weird things happen if you go below the min-width:

exquisite_stew____add-ons_for_firefox

**Comment by tofumatt** [comment]
@muffinresearch oh shoot, sorry I jumped the gun.

I'll check that out and do another PR, I'm sorry.

**Comment by muffinresearch** [comment]
@tofumatt sure thing. I was kind of hoping there'd be a easier way to break of the parent than this but it doesn't seem like that's the case. Or at least not without some more invasive changes. Possibly a quick fix for this will be to use a media query.

**Comment by tofumatt** [comment]
Yeah, I was thinking that too. I'll have a PR soon
- tofumatt

**Pull request merged**